### PR TITLE
06-creating db region table

### LINKED ISSUE #17: 06 - Create table Region
- [x] Create data model schema for Region

### PR DESCRIPTION
**Pull request by Moamin-AbuEwaida**
I'm pushing as a following steps of working on database branch for creating tables.
simple Regions table
this PR has 
region.js file for postgres schema 
create-region.js for sequelize magic to create a table and insert data inside it

### CODE REVIEW

**Comment by dirk-jacobs** [comment]
Tablename in user is in lower-case. Here we have 'tablename: 'Region'
I would make them all lowercase

**Comment by Moamin-AbuEwaida** [comment]
> Tablename in user is in lower-case. Here we have 'tablename: 'Region' I would make them all lowercase

noted @dirk-jacobs

File: api/db/scripts/create-region.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const Region = require('../../models/region.js');
+
+const createRegion = async ()=>{
+    return await Region.bulkCreate([
+        {RegionID: 1, Name:'Brussels'},
```
Comment (rayanejsilva) [inline api/db/scripts/create-region.js:6]: I would add all the 11 provinces: Antwerp, Flemish Brabant, West Flanders, East Flanders, Hainaut, Liège, Limburg, Luxembourg, Namur, Walloon Brabant, Brussels

**Review by rayanejsilva — CHANGES_REQUESTED** [review]
I would add all the 11 provinces

File: api/db/scripts/create-region.js (hunk 1 above)
Comment (Moamin-AbuEwaida) [reply api/db/scripts/create-region.js:6]: > I would add all the 11 provinces: Antwerp, Flemish Brabant, West Flanders, East Flanders, Hainaut, Liège, Limburg, Luxembourg, Namur, Walloon Brabant, Brussels


okaay :+1:  I'll edit the file and recommit it

**Comment by Moamin-AbuEwaida** [comment]
the last commit is adding `const Region = require('../../models/region.js')` into sync.js file

**Review by rayanejsilva — APPROVED** [review]

**Review by Hajir-pro — APPROVED** [review]

**Pull request merged**